docs/Maintain/_category_.json

```diff
@@ -0,0 +1,8 @@
+{
+    "label": "Maintain",
```

Comment:
Maybe it's just me, but "Maintain" for me strikes as "Contribute to our open source code", or something along the lines, and not with Farming, Staking, and Timekeeping. 

Should be keep it simple as "Farming and Staking", or maybe you can think of another 1-word to describe it?